app/templates/list_uploads.html

```diff
@@ -1,9 +1,63 @@
 {% extends "appbuilder/general/model/list.html" %}
-  
+
+{% block head_css %}
+    {{ super() }}
+    <link rel="stylesheet" href="{{url_for('static', filename='css/style.css')}}">
+{% endblock %}
+
+{% block head_js %}
+  {{ super() }}
+  <script src="{{url_for('static', filename='js/zip.min.js')}}"></script>
+  <script src="{{url_for('static', filename='js/submission_check.js')}}"></script>
+{% endblock %}
+
 {% block list_search scoped %}
   <div class="alert alert-info">
     To upload a new submission, please click on the plus icon below.
   </div>
   <hr />
   {{ super() }}
 {% endblock %}
+
+{% block tail_js %}
+    {{ super() }}
+    <script>
+      function init() {
+        var fileInput = document.getElementById("file");
```

Comment:
@Adman So I guess this will work in case there is just one `file` object in there, right?

Reply:
Yes, but ID should be unique throughout the page.